tools/msp_split/splitMSP.xml

```diff
@@ -0,0 +1,39 @@
+<tool id="splitMSP" name="split spectra" version="0.1.0" python_template_version="3.5">
```

Comment:
This tool should also have a creator and maintainer tag as can be seen [here](https://github.com/RECETOX/galaxytools/blob/master/tools/ramclustr/macros.xml)

Reply:
Creators have been added.